### ISSUE TITLE
Web UI settings do not persist

### ISSUE THREAD

**Issue by matejdro** [AndreyPavlenko/transmissionbtc#16]
Steps to reproduce:

1. Enable alternate Web UI
2. Open network settings screen in Web UI:

![image](https://user-images.githubusercontent.com/507922/125774406-e7731dd6-eca7-4a6c-a0d8-9b422714fa11.png)

3. Change some settings and save
4. Restart torrent client (or wait some time)

BUG: Settings will be reset

**Comment by AndreyPavlenko** [comment]
Does it work in the standard Web UI?

**Comment by matejdro** [comment]
Standard web UI has less options:

![image](https://user-images.githubusercontent.com/507922/125820509-1c8156aa-ce70-420a-a7b2-bf1d465e77d8.png)

But from what I see, setting those options work.

**Comment by najevi** [comment]
I agree that persistent settings are desirable but probably not urgent.
I would be happy simply to learn if a settings/config/cookie file exists and then where it might be located.

**Comment by AndreyPavlenko** [comment]
Path to the configuration directory is specified in the app settings and you may change it.
